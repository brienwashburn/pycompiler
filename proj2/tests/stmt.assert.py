assert a
assert True
assert a and b == c
assert foo, bar
