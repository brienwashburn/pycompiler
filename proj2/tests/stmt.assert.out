(Module
 (Assert (Name a))
 (Assert (NameConstant True))
 (Assert
  (BoolOp
   And
   (Name a)
   (Compare (left (Name b)) (ops Eq) (comparators (Name c)))))
 (Assert (Name foo) (Name bar)))
